### PR TITLE
Feature add groceries

### PR DESCRIPTION
**Pull request by rytis17**
Added new tab where you can add groceries with store name and prices.

### CODE REVIEW

**Comment by nocheacosador** [comment]
It think this should have more functionality before it is merged. At least some validation that all of the input fields are filled in. Then store name I think should be a drop down list, etc.

File: ClientApp/src/components/Groceries.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        this.refs.groceriesForm.reset();
+    }
+
+    render(){
+        let groceries = this.state.groceries;
+        return(
+            <div>
+                <form ref="groceriesForm">
+                <label>Store Name</label>
+                <input type="text" ref="storeName"/>
```
Comment (nocheacosador) [inline ClientApp/src/components/Groceries.js]: Here you should use `<Input>` tag provided by `reactstrap`  package. It applies `bootstrap` theme that is global to the project. Same goes for `<button>` tag. You should use `<Button>` instead.

File: ClientApp/src/components/Groceries.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                <label>Item Name</label>
+                <input type="text" ref="itemName"/>
+                <label>Item Price</label>
+                <input type="number" ref="itemPrice"/>
+                <button onClick={e => this.handleSubmit(e)}>Save</button>
+            </form>
+            <table width="500">
```
Comment (nocheacosador) [inline ClientApp/src/components/Groceries.js]: I think `reactstrap` also has its own tag for tables, so you should use it to.

**Review by nocheacosador — CHANGES_REQUESTED** [review]

**Comment by dsemenas** [comment]
You should also add field where it is possible to add date of purchase.

**Review by nocheacosador — APPROVED** [review]
Tight!

**Pull request merged**